### ISSUE TITLE
Having problems in "labelPath" and ""valSet" parameters in "trainCandidates.py"? 

### ISSUE THREAD

**Issue by sugiantonehemia** [cuguilke/microexpnet#7]
Hi, this is a great work! I am interested in tuning the model. 

I am having problems while running "trainCandidates.py". 
python trainCandidates.py -p12 0 label.txt output.jpg outputModel.h5 0.001 1

My questions:
1. Could you please give me an example of the "labelPath" file (the third parameter)?

I tried to make a new file (label.txt) which has content like the following. Is that correct?

> S010_002_00000008.png 1
> S010_002_00000009.png 0

2. What value should I use for valSet (the second parameter)

Thank you in advance!

**Comment by cuguilke** [comment]
Thanks! Let's see...

1. "labelPath" format is like:
>/home/username/Desktop/OuluCASIA/batch0/example.jpeg 0
>/home/username/Desktop/OuluCASIA/batch1/example2.jpeg 3
>...
>/home/username/Desktop/OuluCASIA/batch9/example13.jpeg 2

Note that, "deployImages" function directly uses:
'''
labels.append(int(f.split('/')[6][-1]))
'''
to get the batch no which is then used to decide whether the image belongs to the training or validation set. Therefore, you should modify that part of the code for your case.

2. "valSet" is an integer in [0..9] which corresponds to the batch no given in the image path (for example: 0 in "batch0" part). 

An example trainCandidates.py call would be like:
>python trainCandidates.py -v 9 ./labels_CK.txt ./Graphs/candidateV.png ./Models/tfmodel_candidateV 0.0001 16

One more note: We have used the default Tensorflow model to export the network, so it will produce 4 files:
* outputModelName.data-00000-of-00001
* outputModelName.meta
* outputModelName.index
* checkpoint
